skllm/models/gpt_zero_shot_clf.py

```diff
@@ -19,6 +19,18 @@
 
 
 class _BaseZeroShotGPTClassifier(ABC, BaseEstimator, ClassifierMixin, _OAIMixin):
+    """
+    A base class for zero-shot classification using GPT-3.
+
+    Initialization Parameters
+    ----------
+    openai_key : str, optional
+        The OPEN AI key to use. Defaults to None.
+    openai_org : str, optional
+        The OPEN AI organization to use. Defaults to None.
```

Comment:
I would change this to `The OPEN AI organization ID`

Reply:
I see, that's still a string right?